tests/factories.py

```diff
@@ -142,6 +142,16 @@ class ApiOAuth2ApplicationFactory(ModularOdmFactory):
     callback_url = 'http://example.uk'
 
 
+class ApiOAuth2PersonalTokenFactory(ModularOdmFactory):
+    FACTORY_FOR = ApiOAuth2PersonalToken
+
+    owner = SubFactory(UserFactory)
+
+    scopes = ['osf.full_write']
```

Comment:
According to a spiffy new source code client, the `scopes` field takes a space-delimited string (not a list)...?